src/components/layout/Quiz/index.js

```diff
@@ -0,0 +1,77 @@
+import React from "react"
+import { Redirect } from "react-router-dom"
+import { connect } from "react-redux"
+import PropTypes from "prop-types"
+
+import Compass from "../../../images/compass.png"
+
+// Redux
+import { setLocation } from "../../../actions/quizActions"
+
+// Material UI
+import {
+  Box,
+  Button,
+  CardMedia,
+  Container,
+  Typography,
+} from "@material-ui/core"
+
+// Components
+import StateQuestion from "./StateQuestion"
+
+const Quiz = ({ quiz, setLocation }) => {
+  if (!quiz.agreement) {
+    return <Redirect to="/" />
+  }
+
+  return (
+    <Container maxWidth="lg">
+      <Box display={"flex"} alignItems={"center"} mb={2}>
+        <CardMedia src={Compass}>
+          <img src={Compass} />
+        </CardMedia>
+
+        <Box ml={2}>
+          <Typography variant={"h1"}>Compensation Compass</Typography>
+        </Box>
+      </Box>
+      {quiz.question == 0 ? <StateQuestion /> : null}
+
+      <Box
+        display={"flex"}
+        justifyContent={"space-between"}
+        mt={4}
+      >
+        <Button
+          color="primary"
+          disabled={quiz.question == 0}
+          // TODO: write an onChange method that goes to the previous question
+        >
+          Back
+        </Button>
+
+        <Button
+          color="primary"
+          disabled={quiz.question == 6} // TODO: figure out the right condition here
```

Comment:
A constant could be particularly helpful here, like `quiz.question == LAST_QUESTION` so that if questions are added or removed, we don't have to update all places in the code that depend on how many questions there are. This could be the only place we need to know that, though, so may be over optimizing right now.

Reply:
No I 100% agree with you. I'm going to leave it like this for now, but keep the TODO and make sure that somebody cleans it up (that somebody could be me hehe)